Library/Homebrew/test/gpg_spec.rb

```diff
@@ -12,8 +12,15 @@
 
         shutup do
           subject.create_test_key(dir)
+          gpg = subject::GPG_EXECUTABLE
+          @version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
```

Comment:
I am a bit unsatisfied with these two lines, the implementation is not very good.